### ISSUE TITLE
Finalize feature flags & environments decisions and process

### ISSUE THREAD

**Issue by ecayer** [GSA/notifications-admin#2174]
Finish documentation:

Environments.
When we will remove them.
Are we keeping a list of what the current flags are and what they're used for? Should we? 

Existing docs:
https://github.com/GSA/notifications-admin/blob/main/docs/feature-flagging.md
https://github.com/GSA/notifications-api/blob/main/docs/all.md#feature-flagging

**Comment by heyitsmebev** [comment]
@alexjanousekGSA Do we have a centralized place for all feature flags? Is it `def inject_feature_flags()`: ? 

We could create a dictionary in `create_app` or `feature_flags`.py where all feature flags are listed. Unless PM or other team members need visibility to the list. 

Suggestions for the removal process: 
1. The feature has been QA and works as expected. 
2. Delete all feature_flag conditions 
3. Test the application after removing feature_flag to ensure everything works as expected.
4. Remove any feature_flag tests 
5. Coordinate with other team members any new features ready to go live.

**Comment by ecayer** [comment]
I don't think we need additional visibility into the list at this point. Thanks, @heyitsmebev !
